server/src/Api/Handlers.hs

```diff
@@ -178,6 +176,14 @@ evalTxExecutionUnits cbor =
               , exUnitsSteps = C.executionSteps exUnits
               }
 
+plutusHash :: HashBytesRequest -> AppM ByteStringHash
+plutusHash (HashBytesRequest meth (BytesToHash hs)) = 
+  let hf = case meth of 
```

Comment:
I think possibly hasn't been formatted? In the future, you can run `make format` from the `server` directory (after entering the Haskell dev shell with `nix develop .#hsDevShell`). You can also run `make format` from the root of the repo to format the Purescript code

We have a CI check for formatting at any rate, so if you'll be contributing a lot I'll give you write access and you'll see if it fails due to formatting

Reply:
makes sense, I was running make format always from the root, thanks